src/gtk3.mk

```diff
@@ -21,7 +21,7 @@ endef
 
 define $(PKG)_BUILD
     # Erase post-install script, since it throws errors
-    echo > '$(SOURCE_DIR)/build-aux/meson/post-install.py'
+    echo '#!/bin/true' > '$(SOURCE_DIR)/build-aux/meson/post-install.py'
```

Comment:
That's a funky workaround, but ok :)

I'm actually asking Xavier Claessens if there is a better way.

Reply:
But.. it works 👍🏽

Reply:
@v1993 According to Xavier Claessens from GTK, we need to set `DESTDIR` environment variable during meson install for cross-compilation. This should also skip the post-install stuff: https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/build-aux/meson/post-install.py#L8

Reply:
(I said in a reply that we are using the `--prefix` path, so we do not need to use the DESTDIR. Hopefully we comes with a better solution)

Reply:
Or maybe we SHOULD set DESTDIR=.. ?

Reply:
Emmanuele Bassi  wrote:

> Your cross-compilation suite should set the installation prefix to the final location where the builds must be deployed, and DESTDIR to an intermediate staging location; then, once you built every module in your list, your cross-compilation suite should move all the installed files to their final location, and call the native tools to perform things like updating the installed icons; the desktop files; the MIME database; and the settings schemas.